package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "go-ipfs",
+  "version": "1.0.0",
+  "dependencies": [
+    {
+      "name": "dir-index-html",
+      "hash": "QmeNXKecZ7CQagtkQUJxG3yS7UcvU6puS777dQsx3amkS7"
+    }
+  ]
+}
```

Comment:
cool! yay gx \o/

@whyrusleeping @rht this will "just work" for people without gx since it's checked into the `vendor/` dir, right?  (in the future we can see about not having to vendor in-repo (and just vendor via ipfs) once gx+ipfs proves ultra reliable + intuitive for users) 

we _could_ bite the bullet and abandon `go get` and have the makefile install `gx` and fetch the things for the user. what do we want here for now?